### ISSUE TITLE
Some issues with a blend file

### ISSUE THREAD

**Issue by Akira1San** [UPBGE/blender#256]
Link to file: https://drive.google.com/open?id=0B6yNvVGWiWysa1gtcmQ2cVgySG8
The blend file is a modified version from blendswap. I have some issues:
When AA samples set to off and p key press,  you get black screen on stand alone and embedded. When AA sample 2x, it works. When you disable all the filters, all works.
When you set HDR to full and maximize the viewport and p key, the logic is 70%, is that normal?

**Comment by panzergame** [comment]
Confirmed, in recent master.

**Comment by panzergame** [comment]
@Akira1San : you have an error in one of your GLSL script: CHROMATIC ABERRATION.glsl
```
- vec2 texcoord = gl_TexCoord[0];
+ vec2 texcoord = gl_TexCoord[0].st;
```

**Comment by panzergame** [comment]
Fixed in 36679572e2aec72abb6b01c6509ed0c46621bdcb
